syntax = "proto2";

import "hedwig/protobuf/options.proto";

package tests.bad1;

// error: bad name
message DeviceCreated {
  option (hedwig.message_options).major_version = 1;

  required int32 foobar = 1;
}
